app/src/main/java/net/skyscanner/backpack/demo/compose/SingleSelectChipGroupStory.kt

```diff
@@ -0,0 +1,142 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.demo.compose
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import net.skyscanner.backpack.compose.chip.BpkChipStyle
+import net.skyscanner.backpack.compose.icon.BpkIcon
+import net.skyscanner.backpack.compose.singleselectchipgroup.BpkSingleSelectChipGroup
+import net.skyscanner.backpack.compose.singleselectchipgroup.SingleSelectChipGroupData.ChipGroupType
+import net.skyscanner.backpack.compose.singleselectchipgroup.SingleSelectChipGroupData.ChipItem
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.compose.tokens.Deals
+import net.skyscanner.backpack.compose.tokens.Heart
+import net.skyscanner.backpack.demo.R
+import net.skyscanner.backpack.demo.components.SingleSelectChipGroupComponent
+import net.skyscanner.backpack.demo.meta.ComposeStory
+
+@Composable
+@SingleSelectChipGroupComponent
+@ComposeStory("Default")
+fun SingleSelectChipGroupStoryDefault(modifier: Modifier = Modifier) =
+    SingleSelectChipGroupDemo(BpkChipStyle.Default, modifier)
+
+@Composable
+@SingleSelectChipGroupComponent
+@ComposeStory("On Dark")
+fun SingleSelectChipGroupStoryOnDark(modifier: Modifier = Modifier) =
+    SingleSelectChipGroupDemo(BpkChipStyle.OnDark, modifier)
+
+@Composable
+@SingleSelectChipGroupComponent
+@ComposeStory("On Image")
+fun SingleSelectChipGroupStoryOnImage(modifier: Modifier = Modifier) =
+    SingleSelectChipGroupDemo(BpkChipStyle.OnImage, modifier)
+
+@Composable
+@SingleSelectChipGroupComponent
+internal fun SingleSelectChipGroupDemo(style: BpkChipStyle, modifier: Modifier = Modifier) {
```

Comment:
yeah i think the default, on dark, on image variants we can have on a single page (just like we do now for single vs rail), so no need to navigate deeper. single vs multi select you can also just set as the name (ie `Single Select Wrap` & `Single Select Rail`) rather than specific navigation items